components/editor/modules/dynamiccomponent/index.js

```diff
@@ -11,6 +11,7 @@ import EditOverlay from './EditOverlay'
 import { SG_DYNAMIC_COMPONENT_BASE_URLS } from '../../../../lib/settings'
 
 import dynamicComponentRequire from './require'
+import Placeholder from './Placeholder'
```

Comment:
```suggestion
import dynamicComponentRequire from './identifiers'
```

I just realised that we also need this in `Templates/index.js`—like `dynamicComponentRequire`—for the preview on the publish page.

I would recommend adding a `editor/modules/dynamiccomponent/identifiers.js` file which does a proxy which always returns the Placeholder that can be used here and in `Templates/index.js`:
```js
const identifiers = new Proxy({}, {
  get: () => {
    return Placeholder
  }
})
export default identifiers
```

Reply:
Yeah this is why I did this `null`stuff in the SG – but I'll concede this is cleaner. :)